src/gitHubIssueFetcher.ts

```diff
@@ -37,20 +37,29 @@ export interface GitHubIssue {
     createdAt: string;
     repository: GitHubRepository;
     url: string;
+    updatedAt: string;
+    labels: { edges: GitHubLabelEdge[] };
 }
 
 interface GitHubRepository {
     nameWithOwner: string;
 }
 
+export interface GitHubLabelEdge { 
+    node: {
+        name: string
```

Comment:
I just ran `npm run lint` and it pointed out some missing semi-colons and extra whitespace. We should probably mention that in our README.

Reply:
I'll run and add to the readme.

Happy to open another ticket to make it so that linting runs in pre-push hooks. I was actually thinking about linting earlier and then forgot when I went to commit

Reply:
For other projects, I have used a pre-commit hook for linting. I like catching that as early in the process as possible. We used husky. If you could create a ticket, that would be much appreciated. Thanks!